bitnami/nginx/templates/deployment.yaml

```diff
@@ -35,6 +35,23 @@ spec:
       {{- include "nginx.imagePullSecrets" . | nindent 6 }}
       {{- if .Values.affinity }}
       affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        {{- if eq .Values.podAffinityPreset "soft" }}
+        podAffinity: {{- include "common.affinity.pod.soft" (dict "component" "nginx" "context" $) | nindent 10 }}
+        {{- else if eq .Values.podAffinityPreset "hard" }}
+        podAffinity: {{- include "common.affinity.pod.hard" (dict "component" "nginx" "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if eq .Values.podAntiAffinityPreset "soft" }}
+        podAffinity: {{- include "common.affinity.pod.soft" (dict "component" "nginx" "context" $) | nindent 10 }}
+        {{- else if eq .Values.podAntiAffinityPreset "hard" }}
+        podAffinity: {{- include "common.affinity.pod.hard" (dict "component" "nginx" "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if eq .Values.nodeAffinityPreset "soft" }}
+        nodeAffinity: {{- include "common.affinity.node.soft" (dict "key" "FOO" "values" (list "BAR" "BAZ")) | nindent 10 }}
```

Comment:
Could you add some examples of what could be `FOO` and `BAR`, sorry I can imagine what values could it be?

Reply:
This depends on you k8s nodes labels...

Reply:
then we shouldn't add a preset for nodeAffinity, right?

Reply:
Correct.. Check new implementation